server/routes/users.js

```diff
@@ -37,6 +39,41 @@ router.post("/register", validateRegister, async (req, res) => {
     email: user.email,
   };
 
+  const send = async (address) => {
+    let config = {
+      headers: {
+        Authorization: `Bearer ${process.env.sendgridKey}`,
+      }
+    }
+    let data = {
+      personalizations: [
+        {
+          to: [
+            {
+              email: `${address}`,
+            },
+          ],
+        }
+      ],
+      from: {
+        email: "mentionscrawler123@gmail.com",
+        name: "Mentionscrawler Team"
+      },
+      subject: "Welcome to MentionsCrawler!",
+      content: [{
+        type: "text/plain",
+        value: "Thank you for signing up!"
+      }]
+    }
+    try {
+      await axios.post("https://api.sendgrid.com/v3/mail/send", data, config)
```

Comment:
So SendGrid is done by hitting an API and not by using a node library?

Reply:
there's a library that you still have to install for some reason, but yes oddly it's a post request to sendgrid that does the actual sending